src/comgr.cpp

```diff
@@ -125,6 +138,7 @@ static inline auto to_string(const std::size_t& v) { return std::to_string(v); }
 #define CASE_TO_STRING(id) \
     case id: return #id
 
+/// \todo Request comgr to expose this stuff via API.
```

Comment:
This seems reasonable to me, as does moving the Comgr RAII wrapper for C++ into the same repo as Comgr.

Reply:
That's why I've asked you to review this stuff ;)

BTW The next task is implementing online assembly and device-only (!) HIP compilation. Thanks for reviewing.